### PR TITLE
Added bot heartbeat

### PR DESCRIPTION
**Pull request by BrandonMurnane**

### CODE REVIEW

File: api/v1/controllers/bots.js [hunk 1]
```diff
@@ -191,4 +191,16 @@ module.exports = {
     .catch((err) => u.handleError(next, err, helper.modelName));
   },
 
+  heartbeat(req, res, next) {
+    const authToken = req.headers.authorization;
+    const timestamp = req.body.currentTimestamp;
+    let botName;
```
Comment (kriscfoster) [inline api/v1/controllers/bots.js]: Is botName used anywhere?

File: api/v1/controllers/bots.js [hunk 2]
```diff
@@ -191,4 +191,16 @@ module.exports = {
     .catch((err) => u.handleError(next, err, helper.modelName));
   },
 
+  heartbeat(req, res, next) {
+    const authToken = req.headers.authorization;
```
Comment (kriscfoster) [inline api/v1/controllers/bots.js]: Is this used?

**Comment by BrandonMurnane** [comment]
@kriscfoster changes made

**Comment by BrandonMurnane** [comment]
@iamigo could you look at this its very important for us right now

**Pull request merged**